base_import_xlsx/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Base import Excel",
+    "version": "13.0.0.0.0",
```

Comment:
It's common to start at `1.0.0` not at `0.0.0` (probably a debate that goes back to the start of computing, but lets not go into that here)